blog/content/post/2019-06-21-rte-new-version-and-kit.md

```diff
@@ -0,0 +1,128 @@
+
+---
+
+## Intro
+
+Today we are happy to announce the new version of Remote Testing Environment
+product - welcome RTE `v1.1.0`!
+
+After using RTE over a year at 3mdeb in everyday tasks regarding validation and
+developing firmware for various platforms, we have noticed more and more missing
+features in the presented Orange Pi Hat. Furthermore, we have received valuable
+feedback from our previous customers and we have addressed all requested
+hardware and software upgrades. In result, we have achieved product that is even
+more universal and capable of controlling and flashing wider range of computer
+platforms.
+
+## What's new?
+
+Brand new RTE (previously `v1.0.0`) has been improved mostly from the hardware
+perspective:
+
+* SPI `Vcc` pin has been populated - power is now supplied to the SPI `Vcc`
+  connector
+* RTE is now compatible with 1.8V logic levels
+* user can enable/disable SPI `Vcc` on demand:
+
+| GPIO406 (OC_OUT1) state | SPI Vcc               |
+|:-----------------------:|:---------------------:|
+| 0 - low                 | disabled (by default) |
+| 1 - high                | enabled               |
+
+* user can choose the voltage level for `Vcc` SPI - either 1.8 or 3.3 V:
+
+| GPIO405 (OC_OUT2) state | SPI Vcc voltage level |
+|:-----------------------:|:---------------------:|
+| 0 - low                 | 1.8 V (by default)    |
+| 1 - high                | 3.3 V                 |
+
+* user can enable/disable SPI lines (some platforms have problems when booting
+  with SPI wires connected):
+
+| GPIO404 (OC_OUT3) state | SPI lines (MOSI/MISO/CS/SCLK) |
+|:-----------------------:|:-----------------------------:|
+| 0 - low                 | disabled (by default)         |
+| 1 - high                | enabled                       |
+
+* OC buffers GPIO header (J11) is reduced from 12 to 9 pin. accordingly to the
+  `GPIO404-406` new control feature,
+* new UART header (J18) has been added - USB/UART external converters are not
+  required anymore!
+* user can choose whether RS232 DB9 port (J14) or UART header (J18) is enabled
+  for serial communication by setting jumpers on `UART OUTPUT SELECT` header
+  (J16):
+
+| Jumper position (TX and RX) | Serial communication enabled |
+|:---------------------------:|:----------------------------:|
+| RS232 + COM                 | RS232 DB9 port (J14)         |
+| EXT + COM                   | 3.3V UART header (J18)       |
+
+![uart-header](/img/rte-uart-header.jpg)
+
+* eliminated issue with J6 USB port (unreliable detection of USB devices)
+* smaller capacitors were applied near MAX3232 SOIC
+* added LED indicator (D5) for relay state information
+* added easy accessible reset button (SW1) for resetting the RTE board
+
+![reset-button](/img/rte-reset-btn.jpg)
+
+* added `Open hardware` icon on PCB board
+
+![open-hardware-pcb](/covers/rte-new-revision.jpg)
+
+## Upgraded full kit
+
+We came to the conclusion to remove shop kit options for RTE. From now on there
```

Comment:
"We came to the conclusion to remove" Jakoś fleksja mi nie pasuje. Może "We came to the conclusion that shop kit options for RTE are obsolete and should be removed."